README.md

```diff
@@ -6,30 +6,85 @@
 
 ## Setup guide
 
-[//]: # (TODO: Add detailed steps to obtain credentials and configure the module.)
+To use the Trello connector, you must have access to the Trello API through a Trello account and an API key and token generated from it. If you do not have a Trello account, you can sign up for one [here](https://id.atlassian.com/signup?application=trello&continue=https%3A%2F%2Ftrello.com%2Fauth%2Fatlassian%2Fcallback%3FreturnUrl%3D%252F%26display%3DeyJ2ZXJpZmljYXRpb25TdHJhdGVneSI6InNvZnQifQ%253D%253D%26aaOnboarding%3D%26updateEmail%3D%26traceId%3D%26ssoVerified%3D%26createMember%3Dtrue%26jiraInviteLink%3D&display=eyJ2ZXJpZmljYXRpb25TdHJhdGVneSI6InNvZnQifQ%3D%3D). You can generate your Trello API key and token by following the instructions at [Trello API Keys](https://developer.atlassian.com/cloud/trello/guides/rest-api/api-introduction/).
+
+1. Visit https://trello.com/power-ups/admin and create a new powerup
+   ![powerup](https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-trello/refs/heads/main/docs/setup/resources/trello-powerups.png)
+
+2. Enter the required details like:
+   Name of your Power-Up
+   Iframe Connector URL (can be a placeholder URL during development)
+
+   Click “Create” to create your Power-Up.
+
+   ![trelloform](https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-trello/refs/heads/main/docs/setup/resources/trello-form.png)
+
+3. Navigate to 'API Key' and click on 'Generate'
+
+   ![apikey](https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-trello/refs/heads/main/docs/setup/resources/trello-generateKey.png)
+
+   Copy and securely store your API Key.
+
+4. Next to the API Key description, there's a link to the token.
+   Click the link, authorize the Power-Up, and you’ll be redirected to a page displaying your OAuth token.
+
+   ![Key](https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-trello/refs/heads/main/docs/setup/resources/trello-key.png)
+
+   ![allow](https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-trello/refs/heads/main/docs/setup/resources/trello-permission.png)
+
+   Copy and securely store the Token.
+
+   ![token](https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-trello/refs/heads/main/docs/setup/resources/trello-token.png)
 
 ## Quickstart
 
-[//]: # (TODO: Add a quickstart guide to demonstrate a basic functionality of the module, including sample code snippets.)
+To use the Trello connector in your Ballerina application, update the .bal file as follows:
+
+#### Step 1: Import the module
+
+Import the Trello module.
+
+```bash
+import ballerinax/trello;
+```
+
+#### Step 2: Instantiate a new connector
+
+Create a `Config.toml` file and configure the obtained credentials as follows:
+
+```bash
+key = "your_api_key"
+token = "your_api_token"
+isLiveServer = false
```

Comment:
We don't need this variable to run a program using the Trello connector. Check similar places.
```suggestion
```